### PR TITLE
Add Cards for Cisco ITEv7 Chapters 1 and 2

### PR DESCRIPTION
**Pull request by KaraHops**
Add Cards for Cisco ITEv7 Chapters 1 and 2

### CODE REVIEW

File: tools/add_note.rb [hunk 1]
```diff
@@ -35,29 +35,20 @@ def parse_file(file)
   json_contents
 end
 
+#Deep copy is unnecessary
 def get_last_guid(file_json)
-  Marshal.load(Marshal.dump(file_json['notes'].last['guid']))
+  file_json['notes'].last['guid'].dup
```
Comment (thepurplebuffalo) [inline tools/add_note.rb]: Wasn't this already merged in?

File: tools/CompTIA_A+_220-901.json [hunk 2]
```diff
@@ -0,0 +1,161 @@
+{
```
Comment (thepurplebuffalo) [inline tools/CompTIA_A+_220-901.json]: I don't think this file is necessary.

File: tools/add_note.rb (hunk 1 above)
Comment (KaraHops) [reply tools/add_note.rb]: Yes, it was. I am unfamiliar with the process (if one exists) of excluding commits from pull requests.

**Comment by KaraHops** [comment]
Cancelling pull request to clean up my mess

**Pull request closed without merging**